lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
If you would do that, then you would be using it purely for styling and nothing complex like you're doing in `MultilineElement`. It makes sense for `Multiline Element`, 'cause it has a few view logic in it. </3